etcdserver/etcdserverpb/rpc.proto

```diff
@@ -637,6 +638,8 @@ message AuthUserAddResponse {
 
 message AuthUserGetResponse {
   ResponseHeader header = 1;
+
+  repeated string roles = 2;
```

Comment:
Yes, etcdctl can hide the multiple requests from operations. I think packing everything in a single response isn't required for the simplicity.